src/server-manager/cache-manager.ts

```diff
@@ -245,7 +272,15 @@ export class CacheManager {
     if (!this.cacheRoomMemberRawPayload) {
       throw new Error(`${PRE} getRoomMember() has no cache.`)
     }
-    return this.cacheRoomMemberRawPayload.get(roomId)
+    const cacheData = await this.cacheRoomMemberRawPayload.get(roomId)
+    if (!cacheData) {
+      return undefined
+    }
+    const map: PadplusRoomMemberMap = {}
+    for (const property of Object.keys(cacheData)) {
```

Comment:
I think `contactId` is better than `property` in this case.

Reply:
will replace it with memberId.